packages/charting/src/components/VerticalStackedBarChart/VerticalStackedBarChart.base.tsx

```diff
@@ -299,27 +298,26 @@ export class VerticalStackedBarChartBase extends React.Component<
         dots.push(
           <circle
             key={`${index}-${subIndex}-dot`}
-            cx={isNumeric ? xScale(circlePoint.x) : xBarScale(circlePoint.index) + this._additionalSpace}
+            cx={
+              isNumeric
+                ? xScale(circlePoint.x.xAxisPoint as number)
+                : xBarScale(circlePoint.index) + this._additionalSpace
+            }
             cy={yScale(circlePoint.y)}
             onMouseOver={
               isLegendSelected && selectedLegendTitle === item
-                ? this._linehover.bind(this, circlePoint)
-                : this._onStackHover.bind(
-                    this,
-                    circlePoint.x,
-                    find(this.props.data, (point: IVerticalStackedChartProps) => point.xAxisPoint === circlePoint.x)
-                      ?.lineData,
-                  )
+                ? this._lineHover.bind(this, circlePoint)
+                : this._onStackHover.bind(this, circlePoint.x)
```

Comment:
`circlePoint.x`?? you need to send the StackData right? this will break the hover on circle i belive

Reply:
![image](https://user-images.githubusercontent.com/33802398/98070970-4f509a00-1e88-11eb-9672-dcdbc23ca589.png)

Reply:
Nope. It works fine
![image](https://user-images.githubusercontent.com/1230364/98294583-091d3880-1f65-11eb-8eea-87aed75a1140.png)

Reply:
no it isn't
![image](https://user-images.githubusercontent.com/33802398/98312155-1c78e400-1ff7-11eb-99c2-1ff6a0782c4c.png)